controllers/fenceagentsremediation_controller_test.go

```diff
@@ -252,6 +312,32 @@ func createRunningPod(containerName, podName, nodeName string) *corev1.Pod {
 	return pod
 }
 
+func createTerminatingPod(containerName, podName, nodeName string) *corev1.Pod {
+	pod := buildPod(containerName, podName, nodeName)
+	now := metav1.Now()
+	pod.ObjectMeta = metav1.ObjectMeta{Name: pod.Name, Namespace: pod.Namespace, DeletionTimestamp: &now}
+	ExpectWithOffset(1, k8sClient.Create(context.Background(), pod)).To(Succeed())
```

Comment:
@razo7 the offset improves Gomega logs in case an `Expect` in helper functions like this fails. 
See https://onsi.github.io/gomega/#making-assertions-in-helper-functions
(Maybe we can better use `GinkgoHelper()` instead though, which I only just learned about)

Reply:
IIUC `GinkgoHelper()` is helpful for better distinguishing between multiple assertions in one function, and here we have only one.